localization/english/wc_holy_orders_l_english.yml

```diff
@@ -2,6 +2,244 @@
  #########################
  #		Holy Orders		     #
  #########################
+ ### Any Elemental Group ###
+ holy_order_elemental_article:0 "the "
+ holy_order_elemental:0 "Elementals of [TITLE.GetNameNoTierNoTooltip]"
+ ### Air Group ###
+# holy_order_conclave_of_wind_article:0 "the "
```

Comment:
Sometimes doesn't feel as good in the name, because I thought it was used everywhere... turns out I don't see the article being used in the game at all so I'll just uncomment all of them

Reply:
> Sometimes doesn't feel as good in the name, because I thought it was used everywhere... turns out I don't see the article being used in the game at all so I'll just uncomment all of them

Yeah, it's so even in the vanilla.
I would keep it anyway because these are proper names.